corehq/apps/reports/views.py

```diff
@@ -793,31 +793,31 @@ def email_report(request, domain, report_slug, report_type=ProjectReportDispatch
 
     config.filters = filters
 
-    report_id = None
-    if config.report and hasattr(config.report, '_id'):
-        report_id = config.report._id
-
-    body = _render_report_configs(request, [config],
-                                  domain,
-                                  user_id, request.couch_user,
-                                  True,
-                                  lang=request.couch_user.language,
-                                  notes=form.cleaned_data['notes'],
-                                  once=once,
-                                  report_id=report_id)[0].content
+    report = None
+    if config.report:
+        report = config.report
 
     subject = form.cleaned_data['subject'] or _("Email report from CommCare HQ")
+    content = _render_report_configs(
+        request, [config], domain, user_id, request.couch_user, True, lang=request.couch_user.language,
+        notes=form.cleaned_data['notes'], once=once
+    )[0]
 
     if form.cleaned_data['send_to_owner']:
-        send_html_email_async.delay(subject, request.couch_user.get_email(), body,
-                                    email_from=settings.DEFAULT_FROM_EMAIL, ga_track=True,
-                                    ga_tracking_info={'project_space_id': request.domain})
+        email = request.couch_user.get_email()
+        body = render_full_report_notification(request, content, email, report).content
```

Comment:
Don't need to pass in report here or below since this is a once off email. Also `report` isn't a `ReportNotification`. Would make things clearer if the `report` args were renamed to `report_notification`?